Resources/Private/Templates/PluginPreview/Events.html

```diff
@@ -1,154 +1,165 @@
+        </f:case>
+        <f:case value="events2_searchform">
+            <f:render section="renderPidOfSearchResults" arguments="{pidOfSearchResults: pi_flexform_transformed.settings.pidOfSearchResults}"/>
+            <f:render section="renderCategories" arguments="{categories: pi_flexform_transformed.settings.mainCategories}"/>
+        </f:case>
+        <f:case value="events2_searchresults"/>
+        <f:defaultCase>
+            <tr>
+                <td><strong>List type</strong></td>
+                <td>
+                    <f:switch expression="{pi_flexform_transformed.settings.listType}">
+                        <f:case value="listLatest">Latest</f:case>
+                        <f:case value="listToday">Today</f:case>
+                        <f:case value="listWeek">Week</f:case>
+                        <f:case value="listRange">Range</f:case>
+                        <f:defaultCase>All</f:defaultCase>
+                    </f:switch>
+                </td>
+            </tr>
+            <f:render section="renderPidOfDetailPage" arguments="{pidOfDetailPage: pi_flexform_transformed.settings.pidOfDetailPage}"/>
+            <f:render section="renderPidOfManagementPage" arguments="{pidOfManagementPage: pi_flexform_transformed.settings.pidOfManagementPage}"/>
+            <tr>
+                <td><strong>Merge Events same day</strong></td>
+                <td>
+                    <f:if condition="{pi_flexform_transformed.settings.mergeEventsAtSameDay}">
+                        <f:then>
+                            <core:icon identifier="actions-check"/>
+                        </f:then>
+                        <f:else>
+                            <core:icon identifier="actions-message-ok-close"/>
+                        </f:else>
+                    </f:if>
+                </td>
+            </tr>
+            <tr>
+                <td><strong>Merge recurring Events</strong></td>
+                <td>
+                    <f:if condition="{pi_flexform_transformed.settings.mergeRecurringEvents}">
+                        <f:then>
+                            <core:icon identifier="actions-check"/>
+                        </f:then>
+                        <f:else>
+                            <core:icon identifier="actions-message-ok-close"/>
+                        </f:else>
+                    </f:if>
+                </td>
+            </tr>
+            <f:render section="renderCategories" arguments="{categories: pi_flexform_transformed.settings.categories}"/>
+        </f:defaultCase>
+    </f:switch>
+    </tbody>
 </table>
 
 <f:section name="renderCategories">
-	<f:if condition="{categories}">
-		<tr>
-			<td><strong>Categories</strong></td>
-			<td>{categories}</td>
-		</tr>
-	</f:if>
+    <f:if condition="{categories}">
+        <tr>
+            <td><strong>Categories</strong></td>
+            <td>
+                {e2:settingsFormat(content: '{categories}', glue: ',', limit: '10')}
```

Comment:
No quotes needed, if just a fluid variable is used. Limit can be removed, if you change to abnormal high default from VH ;-)

Reply:
Done